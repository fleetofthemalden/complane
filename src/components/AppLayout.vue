<script setup lang="ts">
defineProps<{
  msg: string
}>()
</script>

<template>
  <div class="greetings">
    <div class="w-fit">
      <img
        alt="Complane logo"
        class="logo mx-auto"
        src="@/assets/logo.svg"
        width="125"
        height="125"
      />
      <h1 class="green font-medium text-5xl my-3 text-emerald-500">{{ msg }}</h1>
    </div>

    <h3 class="text-xl">"Complane" to Secretary Pete about a recent bad flight experience</h3>
  </div>
</template>

<style scoped>
.greetings div {
  margin: 0 auto;
}
.greetings h3 {
  text-align: center;
}

@media (min-width: 1024px) {
  .greetings h1,
  .greetings h3 {
    text-align: left;
  }
}
</style>
