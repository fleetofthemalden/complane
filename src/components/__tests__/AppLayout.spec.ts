import { describe, it, expect, beforeEach } from 'vitest'

import { VueWrapper, mount } from '@vue/test-utils'
import AppLayout from '../AppLayout.vue'

const TEST_MSG = 'I like to Complane'

describe('AppLayout', () => {
  let wrapper: VueWrapper<any>

  beforeEach(() => {
    wrapper = mount(AppLayout, { props: { msg: TEST_MSG } })
  })

  it('renders', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('renders the msg property', () => {
    expect(wrapper.text()).toContain(TEST_MSG)
  })
})
