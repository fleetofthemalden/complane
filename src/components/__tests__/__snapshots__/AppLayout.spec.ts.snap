// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AppLayout > renders 1`] = `
VueWrapper {
  "__app": {
    "_component": {
      "name": "VTU_ROOT",
      "render": [Function],
      "setup": [Function],
    },
    "_container": <div
      data-v-app=""
    >
      <div
        class="greetings"
        data-v-6f6437fc=""
      >
        <div
          class="w-fit"
          data-v-6f6437fc=""
        >
          <img
            alt="Complane logo"
            class="logo mx-auto"
            data-v-6f6437fc=""
            height="125"
            src="/complane/src/assets/logo.svg"
            width="125"
          />
          <h1
            class="green font-medium text-5xl my-3 text-emerald-500"
            data-v-6f6437fc=""
          >
            I like to Complane
          </h1>
        </div>
        <h3
          class="text-xl"
          data-v-6f6437fc=""
        >
          "Complane" to Secretary Pete about a recent bad flight experience
        </h3>
      </div>
    </div>,
    "_context": {
      "app": [Circular],
      "components": {
        "transition": {
          "name": "transition",
        },
        "transition-group": {
          "name": "transition-group",
        },
      },
      "config": {
        "compilerOptions": {},
        "errorHandler": undefined,
        "globalProperties": {},
        "isNativeTag": [Function],
        "optionMergeStrategies": {},
        "performance": false,
        "warnHandler": undefined,
      },
      "directives": {},
      "emitsCache": WeakMap {},
      "mixins": [
        {
          "beforeCreate": [Function],
        },
      ],
      "optionsCache": WeakMap {},
      "propsCache": WeakMap {},
      "provides": {},
      "reload": [Function],
    },
    "_instance": {
      "a": null,
      "accessCache": {},
      "appContext": {
        "app": [Circular],
        "components": {
          "transition": {
            "name": "transition",
          },
          "transition-group": {
            "name": "transition-group",
          },
        },
        "config": {
          "compilerOptions": {},
          "errorHandler": undefined,
          "globalProperties": {},
          "isNativeTag": [Function],
          "optionMergeStrategies": {},
          "performance": false,
          "warnHandler": undefined,
        },
        "directives": {},
        "emitsCache": WeakMap {},
        "mixins": [
          {
            "beforeCreate": [Function],
          },
        ],
        "optionsCache": WeakMap {},
        "propsCache": WeakMap {},
        "provides": {},
        "reload": [Function],
      },
      "asyncDep": null,
      "asyncResolved": false,
      "attrs": {},
      "attrsProxy": null,
      "bc": null,
      "bm": null,
      "bu": null,
      "bum": null,
      "c": null,
      "components": null,
      "ctx": {
        "VTU_COMPONENT": {},
      },
      "da": null,
      "data": {},
      "devtoolsRawSetupState": {
        "VTU_COMPONENT": RefImpl {
          "__v_isRef": true,
          "__v_isShallow": false,
          "_rawValue": {},
          "_value": {},
          "dep": undefined,
        },
      },
      "directives": null,
      "ec": null,
      "effect": ReactiveEffect {
        "_depsLength": 4,
        "_dirtyLevel": 0,
        "_runnings": 0,
        "_shouldSchedule": false,
        "_trackId": 1,
        "active": true,
        "allowRecurse": true,
        "deps": [
          Map {
            [Circular] => 1,
          },
          Map {
            [Circular] => 1,
          },
          Map {
            [Circular] => 1,
          },
          Map {
            [Circular] => 1,
          },
        ],
        "fn": [Function],
        "onTrack": undefined,
        "onTrigger": undefined,
        "scheduler": [Function],
        "trigger": [Function],
      },
      "emit": [Function],
      "emitsOptions": null,
      "emitted": null,
      "exposeProxy": null,
      "exposed": null,
      "inheritAttrs": undefined,
      "isDeactivated": false,
      "isMounted": true,
      "isUnmounted": false,
      "m": null,
      "next": null,
      "parent": null,
      "props": {},
      "propsDefaults": {},
      "propsOptions": [
        {},
        [],
      ],
      "provides": {},
      "proxy": {
        "VTU_COMPONENT": {},
      },
      "refs": {
        "VTU_COMPONENT": {},
      },
      "render": [Function],
      "renderCache": [],
      "root": [Circular],
      "rtc": null,
      "rtg": null,
      "scope": EffectScope {
        "_active": true,
        "cleanups": [],
        "detached": true,
        "effects": [
          ReactiveEffect {
            "_depsLength": 4,
            "_dirtyLevel": 0,
            "_runnings": 0,
            "_shouldSchedule": false,
            "_trackId": 1,
            "active": true,
            "allowRecurse": true,
            "deps": [
              Map {
                [Circular] => 1,
              },
              Map {
                [Circular] => 1,
              },
              Map {
                [Circular] => 1,
              },
              Map {
                [Circular] => 1,
              },
            ],
            "fn": [Function],
            "onTrack": undefined,
            "onTrigger": undefined,
            "scheduler": [Function],
            "trigger": [Function],
          },
        ],
        "parent": undefined,
      },
      "setupContext": null,
      "setupState": {
        "VTU_COMPONENT": {},
      },
      "slots": {},
      "slotsProxy": null,
      "sp": null,
      "subTree": {
        "__v_isVNode": true,
        "__v_skip": true,
        "anchor": null,
        "appContext": null,
        "children": null,
        "component": {
          "a": null,
          "accessCache": {},
          "appContext": {
            "app": [Circular],
            "components": {
              "transition": {
                "name": "transition",
              },
              "transition-group": {
                "name": "transition-group",
              },
            },
            "config": {
              "compilerOptions": {},
              "errorHandler": undefined,
              "globalProperties": {},
              "isNativeTag": [Function],
              "optionMergeStrategies": {},
              "performance": false,
              "warnHandler": undefined,
            },
            "directives": {},
            "emitsCache": WeakMap {},
            "mixins": [
              {
                "beforeCreate": [Function],
              },
            ],
            "optionsCache": WeakMap {},
            "propsCache": WeakMap {},
            "provides": {},
            "reload": [Function],
          },
          "asyncDep": null,
          "asyncResolved": false,
          "attrs": {},
          "attrsProxy": null,
          "bc": null,
          "bm": null,
          "bu": null,
          "bum": null,
          "c": null,
          "components": {},
          "ctx": {
            "msg": "I like to Complane",
          },
          "da": null,
          "data": {},
          "devtoolsRawSetupState": {},
          "directives": null,
          "ec": null,
          "effect": ReactiveEffect {
            "_depsLength": 1,
            "_dirtyLevel": 0,
            "_runnings": 0,
            "_shouldSchedule": false,
            "_trackId": 1,
            "active": true,
            "allowRecurse": true,
            "deps": [
              Map {
                [Circular] => 1,
              },
            ],
            "fn": [Function],
            "onTrack": undefined,
            "onTrigger": undefined,
            "scheduler": [Function],
            "trigger": [Function],
          },
          "emit": [Function],
          "emitsOptions": null,
          "emitted": null,
          "exposeProxy": {},
          "exposed": {},
          "inheritAttrs": undefined,
          "isDeactivated": false,
          "isMounted": true,
          "isUnmounted": false,
          "m": null,
          "next": null,
          "parent": [Circular],
          "props": {
            "msg": "I like to Complane",
          },
          "propsDefaults": {},
          "propsOptions": [
            {
              "msg": {
                "0": false,
                "1": true,
                "required": true,
                "type": [Function],
              },
            },
            [],
          ],
          "provides": {},
          "proxy": {
            "msg": "I like to Complane",
          },
          "refs": {},
          "render": [Function],
          "renderCache": [],
          "root": [Circular],
          "rtc": null,
          "rtg": null,
          "scope": EffectScope {
            "_active": true,
            "cleanups": [],
            "detached": true,
            "effects": [
              ReactiveEffect {
                "_depsLength": 1,
                "_dirtyLevel": 0,
                "_runnings": 0,
                "_shouldSchedule": false,
                "_trackId": 1,
                "active": true,
                "allowRecurse": true,
                "deps": [
                  Map {
                    [Circular] => 1,
                  },
                ],
                "fn": [Function],
                "onTrack": undefined,
                "onTrigger": undefined,
                "scheduler": [Function],
                "trigger": [Function],
              },
            ],
            "parent": undefined,
          },
          "setupContext": {
            "attrs": {},
            "emit": [Function],
            "expose": [Function],
            "slots": {},
          },
          "setupState": {},
          "slots": {},
          "slotsProxy": {},
          "sp": null,
          "subTree": {
            "__v_isVNode": true,
            "__v_skip": true,
            "anchor": null,
            "appContext": null,
            "children": [
              {
                "__v_isVNode": true,
                "__v_skip": true,
                "anchor": null,
                "appContext": null,
                "children": [
                  {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "ce": undefined,
                    "children": null,
                    "component": null,
                    "ctx": null,
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": <img
                      alt="Complane logo"
                      class="logo mx-auto"
                      data-v-6f6437fc=""
                      height="125"
                      src="/complane/src/assets/logo.svg"
                      width="125"
                    />,
                    "key": null,
                    "patchFlag": -1,
                    "props": {
                      "alt": "Complane logo",
                      "class": "logo mx-auto",
                      "height": "125",
                      "src": "/complane/src/assets/logo.svg",
                      "width": "125",
                    },
                    "ref": null,
                    "scopeId": "data-v-6f6437fc",
                    "shapeFlag": 1,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "img",
                  },
                  {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": "I like to Complane",
                    "component": null,
                    "ctx": [Circular],
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": <h1
                      class="green font-medium text-5xl my-3 text-emerald-500"
                      data-v-6f6437fc=""
                    >
                      I like to Complane
                    </h1>,
                    "key": null,
                    "patchFlag": 1,
                    "props": {
                      "class": "green font-medium text-5xl my-3 text-emerald-500",
                    },
                    "ref": null,
                    "scopeId": "data-v-6f6437fc",
                    "shapeFlag": 9,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "h1",
                  },
                ],
                "component": null,
                "ctx": [Circular],
                "dirs": null,
                "dynamicChildren": null,
                "dynamicProps": null,
                "el": <div
                  class="w-fit"
                  data-v-6f6437fc=""
                >
                  <img
                    alt="Complane logo"
                    class="logo mx-auto"
                    data-v-6f6437fc=""
                    height="125"
                    src="/complane/src/assets/logo.svg"
                    width="125"
                  />
                  <h1
                    class="green font-medium text-5xl my-3 text-emerald-500"
                    data-v-6f6437fc=""
                  >
                    I like to Complane
                  </h1>
                </div>,
                "key": null,
                "patchFlag": 0,
                "props": {
                  "class": "w-fit",
                },
                "ref": null,
                "scopeId": "data-v-6f6437fc",
                "shapeFlag": 17,
                "slotScopeIds": null,
                "ssContent": null,
                "ssFallback": null,
                "staticCount": 0,
                "suspense": null,
                "target": null,
                "targetAnchor": null,
                "transition": null,
                "type": "div",
              },
              {
                "__v_isVNode": true,
                "__v_skip": true,
                "anchor": null,
                "appContext": null,
                "ce": undefined,
                "children": ""Complane" to Secretary Pete about a recent bad flight experience",
                "component": null,
                "ctx": null,
                "dirs": null,
                "dynamicChildren": null,
                "dynamicProps": null,
                "el": <h3
                  class="text-xl"
                  data-v-6f6437fc=""
                >
                  "Complane" to Secretary Pete about a recent bad flight experience
                </h3>,
                "key": null,
                "patchFlag": -1,
                "props": {
                  "class": "text-xl",
                },
                "ref": null,
                "scopeId": "data-v-6f6437fc",
                "shapeFlag": 9,
                "slotScopeIds": null,
                "ssContent": null,
                "ssFallback": null,
                "staticCount": 0,
                "suspense": null,
                "target": null,
                "targetAnchor": null,
                "transition": null,
                "type": "h3",
              },
            ],
            "component": null,
            "ctx": [Circular],
            "dirs": null,
            "dynamicChildren": [
              {
                "__v_isVNode": true,
                "__v_skip": true,
                "anchor": null,
                "appContext": null,
                "children": "I like to Complane",
                "component": null,
                "ctx": [Circular],
                "dirs": null,
                "dynamicChildren": null,
                "dynamicProps": null,
                "el": <h1
                  class="green font-medium text-5xl my-3 text-emerald-500"
                  data-v-6f6437fc=""
                >
                  I like to Complane
                </h1>,
                "key": null,
                "patchFlag": 1,
                "props": {
                  "class": "green font-medium text-5xl my-3 text-emerald-500",
                },
                "ref": null,
                "scopeId": "data-v-6f6437fc",
                "shapeFlag": 9,
                "slotScopeIds": null,
                "ssContent": null,
                "ssFallback": null,
                "staticCount": 0,
                "suspense": null,
                "target": null,
                "targetAnchor": null,
                "transition": null,
                "type": "h1",
              },
            ],
            "dynamicProps": null,
            "el": <div
              class="greetings"
              data-v-6f6437fc=""
            >
              <div
                class="w-fit"
                data-v-6f6437fc=""
              >
                <img
                  alt="Complane logo"
                  class="logo mx-auto"
                  data-v-6f6437fc=""
                  height="125"
                  src="/complane/src/assets/logo.svg"
                  width="125"
                />
                <h1
                  class="green font-medium text-5xl my-3 text-emerald-500"
                  data-v-6f6437fc=""
                >
                  I like to Complane
                </h1>
              </div>
              <h3
                class="text-xl"
                data-v-6f6437fc=""
              >
                "Complane" to Secretary Pete about a recent bad flight experience
              </h3>
            </div>,
            "key": null,
            "patchFlag": 0,
            "props": {
              "class": "greetings",
            },
            "ref": null,
            "scopeId": "data-v-6f6437fc",
            "shapeFlag": 17,
            "slotScopeIds": null,
            "ssContent": null,
            "ssFallback": null,
            "staticCount": 0,
            "suspense": null,
            "target": null,
            "targetAnchor": null,
            "transition": null,
            "type": "div",
          },
          "suspense": null,
          "suspenseId": 0,
          "type": {
            "__file": "/Users/dixonminnick/Desktop/dev/complane/src/components/AppLayout.vue",
            "__name": "AppLayout",
            "__scopeId": "data-v-6f6437fc",
            "components": {},
            "props": {
              "msg": {
                "required": true,
                "type": [Function],
              },
            },
            "render": [Function],
            "setup": [Function],
          },
          "u": null,
          "uid": 1,
          "um": null,
          "update": [Function],
          "vnode": [Circular],
          "withProxy": null,
        },
        "ctx": [Circular],
        "dirs": null,
        "dynamicChildren": null,
        "dynamicProps": null,
        "el": <div
          class="greetings"
          data-v-6f6437fc=""
        >
          <div
            class="w-fit"
            data-v-6f6437fc=""
          >
            <img
              alt="Complane logo"
              class="logo mx-auto"
              data-v-6f6437fc=""
              height="125"
              src="/complane/src/assets/logo.svg"
              width="125"
            />
            <h1
              class="green font-medium text-5xl my-3 text-emerald-500"
              data-v-6f6437fc=""
            >
              I like to Complane
            </h1>
          </div>
          <h3
            class="text-xl"
            data-v-6f6437fc=""
          >
            "Complane" to Secretary Pete about a recent bad flight experience
          </h3>
        </div>,
        "key": null,
        "patchFlag": 0,
        "props": {
          "msg": "I like to Complane",
          "ref": "VTU_COMPONENT",
        },
        "ref": {
          "f": false,
          "i": [Circular],
          "k": undefined,
          "r": "VTU_COMPONENT",
        },
        "scopeId": null,
        "shapeFlag": 4,
        "slotScopeIds": null,
        "ssContent": null,
        "ssFallback": null,
        "staticCount": 0,
        "suspense": null,
        "target": null,
        "targetAnchor": null,
        "transition": null,
        "type": {
          "__file": "/Users/dixonminnick/Desktop/dev/complane/src/components/AppLayout.vue",
          "__name": "AppLayout",
          "__scopeId": "data-v-6f6437fc",
          "components": {},
          "props": {
            "msg": {
              "required": true,
              "type": [Function],
            },
          },
          "render": [Function],
          "setup": [Function],
        },
      },
      "suspense": null,
      "suspenseId": 0,
      "type": {
        "name": "VTU_ROOT",
        "render": [Function],
        "setup": [Function],
      },
      "u": null,
      "uid": 0,
      "um": null,
      "update": [Function],
      "vnode": {
        "__v_isVNode": true,
        "__v_skip": true,
        "anchor": null,
        "appContext": {
          "app": [Circular],
          "components": {
            "transition": {
              "name": "transition",
            },
            "transition-group": {
              "name": "transition-group",
            },
          },
          "config": {
            "compilerOptions": {},
            "errorHandler": undefined,
            "globalProperties": {},
            "isNativeTag": [Function],
            "optionMergeStrategies": {},
            "performance": false,
            "warnHandler": undefined,
          },
          "directives": {},
          "emitsCache": WeakMap {},
          "mixins": [
            {
              "beforeCreate": [Function],
            },
          ],
          "optionsCache": WeakMap {},
          "propsCache": WeakMap {},
          "provides": {},
          "reload": [Function],
        },
        "children": null,
        "component": [Circular],
        "ctx": null,
        "dirs": null,
        "dynamicChildren": null,
        "dynamicProps": null,
        "el": <div
          class="greetings"
          data-v-6f6437fc=""
        >
          <div
            class="w-fit"
            data-v-6f6437fc=""
          >
            <img
              alt="Complane logo"
              class="logo mx-auto"
              data-v-6f6437fc=""
              height="125"
              src="/complane/src/assets/logo.svg"
              width="125"
            />
            <h1
              class="green font-medium text-5xl my-3 text-emerald-500"
              data-v-6f6437fc=""
            >
              I like to Complane
            </h1>
          </div>
          <h3
            class="text-xl"
            data-v-6f6437fc=""
          >
            "Complane" to Secretary Pete about a recent bad flight experience
          </h3>
        </div>,
        "key": null,
        "patchFlag": 0,
        "props": null,
        "ref": null,
        "scopeId": null,
        "shapeFlag": 4,
        "slotScopeIds": null,
        "ssContent": null,
        "ssFallback": null,
        "staticCount": 0,
        "suspense": null,
        "target": null,
        "targetAnchor": null,
        "transition": null,
        "type": {
          "name": "VTU_ROOT",
          "render": [Function],
          "setup": [Function],
        },
      },
      "withProxy": null,
    },
    "_props": null,
    "_uid": 0,
    "component": [Function],
    "config": {
      "compilerOptions": {},
      "errorHandler": undefined,
      "globalProperties": {},
      "isNativeTag": [Function],
      "optionMergeStrategies": {},
      "performance": false,
      "warnHandler": undefined,
    },
    "directive": [Function],
    "mixin": [Function],
    "mount": [Function],
    "provide": [Function],
    "runWithContext": [Function],
    "unmount": [Function],
    "use": [Function],
    "version": "3.4.21",
  },
  "__setProps": [Function],
  "cleanUpCallbacks": [
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
  ],
  "componentVM": {},
  "isDisabled": [Function],
  "rootVM": {
    "VTU_COMPONENT": {},
  },
  "wrapperElement": <div
    class="greetings"
    data-v-6f6437fc=""
  >
    <div
      class="w-fit"
      data-v-6f6437fc=""
    >
      <img
        alt="Complane logo"
        class="logo mx-auto"
        data-v-6f6437fc=""
        height="125"
        src="/complane/src/assets/logo.svg"
        width="125"
      />
      <h1
        class="green font-medium text-5xl my-3 text-emerald-500"
        data-v-6f6437fc=""
      >
        I like to Complane
      </h1>
    </div>
    <h3
      class="text-xl"
      data-v-6f6437fc=""
    >
      "Complane" to Secretary Pete about a recent bad flight experience
    </h3>
  </div>,
}
`;
